.circleci/build_autoinst.sh

```diff
@@ -18,12 +18,12 @@
 set -ex
 
 destdir=${1:-../os-autoinst}
-sha=${2}
+sha=${2:-./autoinst.sha}
```

Comment:
sha is supposed to contain a shasum, not a filename. did you mean:
```
sha=${2:-$(cat ./autoinst.sha)}
```
?

Reply:
Indeed, that was a thinko on my part. Thanks!